### PR TITLE
updates to Tutorial

### PR DESCRIPTION
**Pull request by BPylypenko**
Updates for fixing inaccuracies in the tutorial

### CODE REVIEW

File: source/tutorials_wine.rst [hunk 1]
```diff
@@ -320,14 +320,14 @@ Paste this code into the file:
...
-      uri: gs://<paste your bucket address here>/data-tutorial/wine-quality.csv
-      region: <paste region here>
-      keySecret: <paste key secret here>
-      description: Wine dataset
+   kind: Connection
+   id: wine-tutorial
+   spec:
+     type: gcs
+     uri: gs://<paste your bucket address here>/data-tutorial/wine-quality.csv
+     project: <paste project name here>
```
Comment (mcdoker18) [inline source/tutorials_wine.rst]: Revert the change because there is no project parameter.

File: source/tutorials_wine.rst [hunk 2]
```diff
@@ -413,28 +413,28 @@ Paste code into the file:
...
+   spec:
+     model:
+       name: wine
+       version: 1.0
+     toolchain: mlflow  # MLFlow training toolchain integration
+     entrypoint: main
+     workDir: mlflow/sklearn/wine  # MLproject location (in GitHub)
+     data:
+       - connName: wine-tutorial
+         localPath: data-totorial/wine-quality.csv # wine-quality.csv file (on GCS)
```
Comment (mcdoker18) [inline source/tutorials_wine.rst]: Why did you make this change? The old path is valid.

**Review by mcdoker18 — CHANGES_REQUESTED** [review]

File: source/tutorials_wine.rst (hunk 2 above)
Comment (BPylypenko) [reply source/tutorials_wine.rst]: I made this change because in our tutorial is the instruction:

image

File: source/tutorials_wine.rst (hunk 1 above)
Comment (BPylypenko) [reply source/tutorials_wine.rst]: Why then we insert in 'region' parameter data from 'project ID' or should I take this information from somewhere else?

Or why this parameter is named 'region'?
image

Comment (mcdoker18) [reply source/tutorials_wine.rst]: Different clouds have different names for the parameter. So we choose one name.
You can find the GCP connection documentation here https://docs.odahu.org/ref_connections.html#google-cloud-storage .

Comment (BPylypenko) [reply source/tutorials_wine.rst]: > Revert the change because there is no project parameter.

Why then we insert in 'region' parameter data from 'project ID' (on GCP) or should I take this information from somewhere else?

Or why this parameter is named 'region'?

File: source/tutorials_wine.rst (hunk 2 above)
Comment (mcdoker18) [reply source/tutorials_wine.rst]: remotePath - file in a GCS bucket
localPath - file in a training container
[documentation](https://docs.odahu.org/ref_trainings.html#training-data)

Comment (BPylypenko) [reply source/tutorials_wine.rst]: localPath: data-totorial/wine-quality.csv # wine-quality.csv file (on GCS)

Then we should change:
localPath: mlflow/sklearn/wine/wine-quality.csv # wine-quality.csv file <s>(on GCS)</s>

**and change description to 446 line:**
- line 12: ``localPath`` relative path of the data file at the training (docker) container where data were put, in our case, from gitHub repository.

File: source/tutorials_wine.rst [hunk 3]
```diff
@@ -612,12 +612,12 @@ Paste code into the file:
...
-    spec:
-      image: "<fill-in>"
-      minReplicas: 1
-      ImagePullConnectionID: docker-tutorial
+   id: wine-tutorial
+   kind: ModelDeployment
+   spec:
+     image: "<fill-in>"
+     minReplicas: 1
+     ImagePullConnectionID: docker-tutorial
```
Comment (mcdoker18) [inline source/tutorials_wine.rst]: `ImagePullConnID` -> 'imagePullConnID'

**Review by mcdoker18 — CHANGES_REQUESTED** [review]
